### ISSUE TITLE
lndltc syncing for ages (stuck with `"synced_to_chain": false,`)

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud-simnet#41]
What I did with native setup:
1. fresh `xud-simnet-clean` and 
2. `xud-simnet-install` 

and lndltc is seemingly stuck syncing:

```
~/xud-simnet$ xud-simnet-start
starting all processes
ltcd started
lndbtc started
lndltc started
raiden started
xud started
sync lndbtc to chain
.
lndbtc synced to chain
sync lndltc to chain
.................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
```

Problem is LND's known bug where it's stuck on `"synced_to_chain": false,` even though it's caught up, in my scenario above you can see the blockheights are equal:

```
kilrau@K-Yoga:~/xud-simnet$ lndltc-lncli getinfo
{
	"version": "0.6.1-beta commit=v0.6.1-beta",
	"identity_pubkey": "02c2bfb01edc15f05cfe38078a19a3dfb886a35092170797c60c8d21466933d22f",
	"alias": "LTC@K-Yoga",
	"num_pending_channels": 0,
	"num_active_channels": 0,
	"num_inactive_channels": 0,
	"num_peers": 0,
	"block_height": 10681,
	"block_hash": "c69effdbed13e3fb4cab52a8ca7c4bbddf942ff74881a03f282b24b576fe7c29",
	"best_header_timestamp": 1563291855,
	"synced_to_chain": false,
	"testnet": false,
	"chains": [
		{
			"chain": "litecoin",
			"network": "simnet"
		}
	],
	"uris": null
}
kilrau@K-Yoga:~/xud-simnet$ ltcctl getinfo
Error creating a default config file: open /home/kilrau/.ltcd/ltcd.conf: no such file or directory
{
  "version": 120000,
  "protocolversion": 70002,
  "blocks": 10681,
  "timeoffset": 0,
  "connections": 1,
  "proxy": "",
  "difficulty": 1,
  "testnet": false,
  "relayfee": 0.00001,
  "errors": ""
}
```

### Proposal:

test lnd 0.7 which is supposed to fix some things here. If it's still happening: do what we do on docker: compare blockheight lndltc with blockheight ltcd, if the same, restart lndltc which fixes it and magically turns` "synced_to_chain": true`.

**Comment by offerm** [comment]
The `block_height` that you see on LND is not the number of blocks that LND processed. It means that LND is aware that there are `block_height` blocks and is now processing these blocks.

If we do that on docker, it can explain why we see the "engine is still syncing" error. 

@kilrau have you seen this problem with native simnet using the latest code?

**Comment by kilrau** [comment]
> The block_height that you see on LND is not the number of blocks that LND processed. It means that LND is aware that there are block_height blocks and is now processing these blocks.

Aware, but LND gets stuck there and never finishes processing and switches to `synced_to_chain=true`. Saw this on native SimNet at the time I opened the issue. If you fixed something on this I can test again. Let me know.

**Comment by offerm** [comment]
In the past the wallet birthday was as the first block. This is the default
for dinner and testnet. This cause a very slow sync time. I changed it to
work like main net
Please try to reproduce

**Comment by kilrau** [comment]
Tested this again. Still the case. Can reproduce this reliably.

What makes lndltc switch to `synced_to_chain=true`:
`lndltc-lncli stop`
`xud-simnet-start`

Let's do https://github.com/ExchangeUnion/xud-simnet/issues/45 and see if the new lnd version fixes it.

**Comment by kilrau** [comment]
This is happening in 100% of the cases for me by now. Bumping up to P1.

**Comment by kilrau** [comment]
I will test this again

**Comment by kilrau** [comment]
Tested, solved with lnd `0.7.1`. Closing.
